.github/actions/integration-test/action.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Run Integration Test"
```

Comment:
This is a more elaborate integration test setup that runs on plain ubuntu github actions. The setup behavior is much more complex than before (or rather - the complexity is explicit, rather than having been done out of sight on ec2 runners), and has been moved to a separate action so as to be callable from multiple locations.